ietfdata/datatracker.py

```diff
@@ -276,6 +276,16 @@ class Submission:
     words           : Optional[int]
 
 
+@dataclass(frozen=True)
+class SubmissionEvent:
+    by              : Optional[PersonURI]
+    desc            : str
+    id              : int
+    resource_uri    : str
```

Comment:
Should this be a `SubmissionEventURI` type? The goal is to move away from URIs as strings.

Reply:
Yes; changed now in 7270acc

Reply:
Thanks – also needs a `submission_event(self, submission_event_uri)` method.

Reply:
Done -- added in the latest commits, along with a ```test_submission_event()``` method in ```test_datatracker.py```.